### PR TITLE
Add MTU Size support and Configuration edit through Helm charts

### PR DESCRIPTION
**Pull request by brecode**
This PR adds support to make MTU size configurable through Helm charts and contiv-vpp.yaml. 

Added MTU field on Af_packet and tap interface configuration to support different MTU sizes. 
Field was added to Pod-VPP and interconnectAfpacket/interconnectTapHost interfaces

### CODE REVIEW

File: k8s/contiv-vpp/templates/vpp.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ data:
     TAPInterfaceVersion: {{ .Values.contiv.tapInterfaceVersion }}
     {{- if .Values.contiv.stealTheNIC }}
     StealTheNIC: True
+    MTUSize: {{ .Values.contiv.mtuSize }}
     {{- end }}
     IPAMConfig:
```
Comment (jmedved) [inline k8s/contiv-vpp/templates/vpp.yaml]: MTUSize needs to move past Line 29

File: plugins/contiv/pod.go [hunk 2]
```diff
@@ -241,6 +245,7 @@ func (s *remoteCNIserver) podTAP(request *cni.CNIRequest, podIPNet *net.IPNet) *
 	return &linux_intf.LinuxInterfaces_Interface{
 		Name:    "pod-" + s.tapTmpHostNameFromRequest(request),
 		Type:    linux_intf.LinuxInterfaces_AUTO_TAP,
+		Mtu:     s.config.MTUSize,
```
Comment (jmedved) [inline plugins/contiv/pod.go]: A question - where is the MTU set for the interface to the Linux stack?

Comment (brecode) [reply plugins/contiv/pod.go]: Yes, check host.go changes

File: k8s/contiv-vpp/templates/vpp.yaml (hunk 1 above)
Comment (brecode) [reply k8s/contiv-vpp/templates/vpp.yaml]: Yup, quick copy-paste mistake

**Comment by rastislavszabo** [comment]
Nikos, please also cover the configuration in https://github.com/contiv/vpp/blob/master/cmd/contiv-init/vppcfg.go - to support the STN case.

**Pull request merged**